config.yaml

```diff
@@ -15,36 +15,85 @@ submodels:
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/share/atmosphere/aerosol/global.N96/2020.05.19/sulpc_oxidants_N96_L38
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/share/atmosphere/aerosol/global.N96/2020.05.19/DMS_conc.N96
 
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/historical/atmosphere/forcing/global.N96/2021.06.22/ozone_1849_2015_ESM1.anc
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/historical/atmosphere/forcing/resolution_independent/2021.06.22/volcts_cmip6.dat
+
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/historical/atmosphere/forcing/resolution_independent/2021.06.22/TSI_CMIP6_ESM_v2
+
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/historical/atmosphere/land/biogeochemistry/global.N96/2021.06.22/Ndep_1849_2015.anc
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/historical/atmosphere/land/vegetation/global.N96/2024.07.04/cableCMIP6_LC_1850-2015.nc
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/share/atmosphere/land/soiltype/global.N96/2020.05.19/qrparm.soil_igbp_vg
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/share/atmosphere/land/vegetation/global.N96/2020.05.19/cable_vegfunc_N96.anc
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/land/biogeochemistry/resolution_independent/2020.05.19/modis_phenology_csiro.txt
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/land/biogeochemistry/resolution_independent/2020.05.19/pftlookup_csiro_v16_17tiles_wtlnds.csv
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/land/biogeophysics/resolution_independent/2020.05.19/def_soil_params.txt
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/land/biogeophysics/resolution_independent/2020.05.19/def_veg_params.txt
+        
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/spectral/resolution_independent/2020.05.19/spec3a_sw_hadgem1_6on
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/spectral/resolution_independent/2020.05.19/spec3a_lw_hadgem1_6on
+
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/modern/share/atmosphere/grids/global.N96/2020.05.19/qrparm.mask
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/grids/resolution_independent/2020.05.19/vertlevs_G3
+
+        - /g/data/tm70/sw6175/esm1p5-input-restructure/restructured-inputs/share/atmosphere/stash/2020.05.19/
```

Comment:
Same comment as other review: if the groupings are meaningful maybe use comments rather than blank lines.

Reply:
These aren't using the `vk83` location. Should they be?

Reply:
Oh good catch, my bad! I've fixed this up but will update the paths with the new `configurations` directory before requesting a new review.